packages/appeals-service-api/src/services/appeal.service.js

```diff
@@ -141,6 +150,74 @@ async function updateAppeal(id, appealUpdate) {
 	return updatedAppeal;
 }
 
+/**
+ * @param {*} appealUpdate - appeal to update
+ * @param {*} appeal - existing appeal
+ * @returns {Promise<void>}
+ */
+async function linkToUser(appealUpdate, appeal) {
+	const updateLinkForS76 =
+		appealUpdate?.contactDetailsSection?.isOriginalApplicant !== undefined &&
+		(appeal?.contactDetailsSection?.isOriginalApplicant === undefined ||
+			appealUpdate?.contactDetailsSection?.isOriginalApplicant !==
+				appeal?.contactDetailsSection?.isOriginalApplicant);
```

Comment:
I find this hard to follow with the nesting. What about this? (similar for HAS)

even this is a bit long - they're awkward long names!

```
const contactDetailsIsOriginalApplicant = appeal?.contactDetailsSection?.isOriginalApplicant;
const updateContactDetailsIsOriginalApplicant = appealUpdate?.contactDetailsSection?.isOriginalApplicant;
const updateLinkForS76 = updateContactDetailsIsOriginalApplicant  !== undefined &&
		(updateContactDetailsIsOriginalApplicant  !== contactDetailsIsOriginalApplicant );
```